magicctapipe/io/__init__.py

```diff
@@ -6,24 +6,28 @@
     RealEventInfoContainer,
     SimEventInfoContainer,
 )
-from .gadf import (
-    create_event_hdu,
-    create_gh_cuts_hdu,
-    create_gti_hdu,
-    create_pointing_hdu,
-)
 from .io import (
+    telescope_combinations,
     format_object,
     get_dl2_mean,
     get_stereo_events,
+    get_stereo_events_old,
     load_dl2_data_file,
     load_irf_files,
     load_lst_dl1_data_file,
     load_magic_dl1_data_files,
     load_mc_dl2_data_file,
     load_train_data_files,
+    load_train_data_files_tel,
     save_pandas_data_in_table,
 )
+from .gadf import (
+    create_event_hdu,
+    create_gh_cuts_hdu,
+    create_gti_hdu,
+    create_pointing_hdu,
+)
+
 
```

Comment:
It is better to keep import source in alphabetical order

Reply:
Done.

Reply:
I think the comment here was that "gadf" should be before "io" as it was before

Reply:
But the io 'telescope combination' is imported in gadf, and the code will crash if we switch the modules order (we will need to duplicate again the function in gadf)

Reply:
is this still the case? in `gadf` I see the import of `telescope_combinations` is commented.

Reply:
Actually we use it in the new version of create_event_hdu, which is not needed in this PR but will be needed in the next one. So I commented this line as now it is not used (and pyflakes will complain) but I kept the __init__ ordering not to change it again in few weeks/days as we will need this import in the next PR